test/cljs/rems/test_fields.cljs

```diff
@@ -20,9 +20,11 @@
 (deftest field-wrapper-test
   ;; TODO: experiment of writing tests for view components; is there better tooling for testing components?
   (testing "link to show diff"
-    (testing "no previous value"
-      (is (not (contains-hiccup? toggle-diff-button (field-wrapper {:form/id "foo" :field/id "foo" :field/value "foo"} "<editor-component>")))))
-    (testing "has previous value"
-      (is (contains-hiccup? toggle-diff-button (field-wrapper {:form/id "foo" :field/id "foo" :field/value "foo", :field/previous-value "bar"} "<editor-component>"))))
-    (testing "previous value is same as current value"
-      (is (not (contains-hiccup? toggle-diff-button (field-wrapper {:form/id "foo" :field/id "foo" :field/value "foo" :field/previous-value "foo"} "<editor-component>")))))))
+     (testing "no previous value"
+        (is (not (contains-hiccup? toggle-diff-button 
+                                   (field-wrapper {:form/id "foo" :field/id "foo" :field/value "foo"} "<editor-component>")))))
+     (testing "has previous value"
+       (is (contains-hiccup? toggle-diff-button 
+                             (field-wrapper {:form/id "foo" :field/id "foo" :field/value "foo", :field/previous-value "bar"} "<editor-component>"))))
+     (testing "previous value is same as current value"
+       (is (not (contains-hiccup? toggle-diff-button (field-wrapper {:form/id "foo" :field/id "foo" :field/value "foo" :field/previous-value "foo"} "<editor-component>")))))))
```

Comment:
What is the change here?

Reply:
hmm, no change really

Reply:
Ok just a formatting change I guess (split line + indent)